python-avd/pyavd/_eos_designs/structured_config/underlay/prefix_lists.py

```diff
@@ -34,6 +34,9 @@ def prefix_lists(self: AvdStructuredConfigUnderlay) -> list | None:
         if not self.shared_utils.underlay_filter_redistribute_connected:
             return None
 
+        if not self.shared_utils.underlay_ipv4:
```

Comment:
There are other prefix lists getting created here. Ex. the PIM RP stuff. If underlay multicast is not supported in combination with v6, we should either document that clearly in the schema or even better check for it and raise if we get this unsupported combination.
Same for WAN, it seems like v6 would not work for that, so we need to document or catch that.